### ISSUE TITLE
First restoring taskbar before restoring (full screen) apps?

### ISSUE THREAD

**Issue by Stefan1200-de** [kangyu-california/PersistentWindows#246]
While I work at home, I have an open RDP connection (using MSTSC) to my office.
Persistent Windows (current version) seems to restore the MSTSC app first, before it tries to restore my taskbar. This does not work, because the full screen app MSTSC is above my taskbar.
Is it possible to restore the taskbar first? I guess this should fix this issue.

Using Windows 10 22H2.

**Comment by kangyu-california** [comment]
the issue is mitigated by restore full-screen mstsc window in the second restore pass.
[PersistentWindows5.43_0812.zip](https://github.com/kangyu-california/PersistentWindows/files/12328561/PersistentWindows5.43_0812.zip)

Most of the time, Windows OS is taking charge of restoring taskbar, and PW is just watching, as evidenced by "PersistentWindows: avoid restore unresponsive window $taskbar$"

**Comment by Stefan1200-de** [comment]
Thanks, I already installed that version 3 days ago. Now restoring the taskbar was working every time.

But something off while restoring MSTSC. It feels like Persistent Windows only see/save MSTSC in windowed mode (I work only with maximized or completely minimized MSTSC), but it seems it maximizes MSTSC sometimes, if I click on it on my taskbar. Sometimes it restores it after restoring the taskbar, without I have to click anything. Sometimes, as soon as I switch back to my local desktop while minimizing MSTSC, it change MSTSC to window mode. Currently I can't describe it better, I keep an eye on it and give more information after some more work days.

**Comment by Stefan1200-de** [comment]
Played around a bit today.
Today it maximizes MSTSC every time. After around one minute I minimized MSTSC to work with another local application.
Short time later I maximized MSTSC again. Now MSTSC was only in a small window, but I never switched MSTSC in window mode. And I had the feeling that something was controlling my mouse pointer for a short time (maybe one second).

Could be this the effect of Persistent Windows 5.43_0812?
I will disable Persistent Windows for some more tests.
